SPECS/ntopng/ntopng.spec

```diff
@@ -1,18 +1,22 @@
-%global nDPIver 4.2
+%global nDPIver 4.8
+%global hiredisVer 1.2.0
 Summary:        Web-based Network Traffic Monitoring Application
 Name:           ntopng
-Version:        5.2.1
-Release:        2%{?dist}
+Version:        6.0
+Release:        1%{?dist}
 License:        GPLv3
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Applications/System
 URL:            https://www.ntop.org/
-#Source0:       https://github.com/ntop/ntopng/archive/%{version}.tar.gz
+#Source0:       https://github.com/ntop/ntopng/archive/refs/tags/%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
-#Source1:       https://github.com/ntop/nDPI/archive/%{nDPIver}.tar.gz
+#Source1:       https://github.com/ntop/nDPI/archive/refs/tags/%{nDPIver}.tar.gz
 Source1:        nDPI-%{nDPIver}.tar.gz
+# Source2:      https://github.com/redis/hiredis/archive/refs/tags/v%{hiredisVer}.tar.gz
+Source2:        hiredis-%{hiredisVer}.tar.gz
 Patch1:         CVE-2021-45985.patch
```

Comment:
is this patch still needed?

Reply:
Yes it is. In the upstream repo, the file that is affected by this patch hasn't been updated for 3 years.